src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
   // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map((item) => item.split(':').map((part) => part.trim()))
+    .filter((item) => item.length === 2 && item[0] && item[1]);
+
+  const stylesObject = {};
+
+  stylesArray.forEach(([key, value]) => {
+    stylesObject[key] = value;
+  });
+
+  return stylesObject;
```

Comment:
The function correctly converts the string to an object representation of the CSS styles. The use of `forEach` to iterate over the array and construct the object is appropriate here.